.github/workflows/archive.yml

```diff
@@ -1,22 +1,20 @@
 # This is a basic workflow to help you get started with Actions
```

Comment:
We have a bunch of default comments that can probably be cleaned up.

Reply:
agreed! i never paid any attention to those :)
it looks a lot better without them!